test/check-application

```diff
@@ -508,6 +518,31 @@ class TestFiles(testlib.MachineCase):
         b.click(".breadcrumb-button:nth-of-type(3)")
         m.execute("sudo chattr -i /home/admin/newdir")
 
+        b.go("/files#/?path=/root")
+        b.wait_not_present(".pf-v5-c-empty-state")
+        try:
+            self.create_directory(b, "newdir")
+            b.wait_visible("[data-item='newdir']")
+            self.assert_owner('/root/newdir', 'root:root')
+        finally:
+            m.execute(['rm', '-rf', '/root/newdir'])
+
+        m.execute('useradd testuser')
+        b.go('/files#/?path=/home/testuser')
+        b.wait_not_present('.pf-v5-c-empty-state')
```

Comment:
On Arch this fails because `useradd` does not create a home directory. It is not configured to do so per `/etc/default/useradd`.

So likely you want to pass `-m`